include/dfu/dfu_target.rst

```diff
@@ -71,6 +71,7 @@ Full modem upgrades
    An external flash memory device of minimum 2MB is required for this target.
 
 This type of firmware upgrade supports updating the modem firmware using the serialized firmware bundled in the zip file of the modem firmware release.
+The serialized firmware file uses the :file:`.cbor` extension.
```

Comment:
@hakonfam Doesn't it support also .bin files? The sample is using .bin files right now.